### PR TITLE
Compatibility: Add clarity about markdown pages and image optimization

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,9 +11,11 @@ Tuesday UTC may not be incorporated due to time constraints.
 
 The compatibility matrix section of the website is built from
 [YAML](https://yaml.org/) files located in [_data/compatibility/](_data/compatibility/).
-The compatibility images (usability screenshots, logos) are located in
-[img/compatibility/](img/compatibility/) with sub-folders for each
-wallet or service. These files are free for anyone to repurpose/republish
+Each wallet also requires a markdown file in
+[en/compatibility/](en/compatibility/). The compatibility images (usability
+screenshots, logos) are located in [img/compatibility/](img/compatibility/) with
+sub-folders for each wallet or service. Make sure to optimize any images using
```
Comment (jnewbery) [inline CONTRIBUTING.md]: s/optimize and images/optimize and png files/

**Comment by jnewbery** [comment]
Thanks for opening this, @bitschmidty

File: CONTRIBUTING.md (hunk 1 above)
Comment (bitschmidty) [reply CONTRIBUTING.md]: Changed!

**Comment by jnewbery** [comment]
ACK d32b43cb06814f5a90c2fd74f5503e2db7a4797e

**Comment by harding** [comment]
LGTM.  Thanks!

**Pull request merged**